packages/react-tabs/Spec.md

```diff
@@ -316,51 +403,79 @@ export type TabListProps = ComponentProps<TabListSlots> & {
 
 ### From v0
 
-- Replace use of `<Menu>` with `<TabList>` (full rewrite)
+1. Replace use of `<Menu>` with `<TabList>`
+2. Replace items data with Tab instances, writing a `map` function as needed.
 
 ## Behaviors
 
 ### Selection
 
-The TabList will provide single-selection of a tab.
+TabList provides single selection of a tab.
 
-### Programmatic Selection
+#### **Programmatic**
 
 The TabList.selectedKey can control the currently selected tab.
-If the tab to be selected is in the overflow, it will be moved into the set of visible tabs.
-
-> TODO: Do we need an announce here?
+The onTabSelected event informs callers of a selection change.
+The event data includes the tab value.
 
-### Overflow
+#### **Mouse/Touch**
 
-The overflow part will screen read as a button with text indicating to click for a list of other available tabs.
+Clicking the tab selects it.
 
-### Mouse/Touch
+#### **Keyboard**
 
-Clicking the tab will raise the TabsList.onTabSelected event
+The tab list will support the prescribed [ARIA keyboard interaction](https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-19).
 
-### Keyboard
+A Tablist allows focus of individual tabs.
 
-TabsList will focus trap on the set of tabs.
-When focused on the tabs, the user can cycle through tabs with arrow keys.
-The arrow keys are based on layout. Horizontal: left/right, Vertical: up/down
-The TAB key moves focus from tabs to overflow to next control in the tab order.
+When the TabList has focus, the user can use arrow keys to move focus to the next or previous tab.
+The keys are based on layout. Horizontal uses left/right arrows and vertical uses up/down arrows.
 
-> TODO: Should Ctrl+Tab and Shift+Ctrl+Tab be supported?
+Whe the TabList has focus, the user can use the home/end keys to move focus to the first/last tab.
 
-### Screen readers
+A tab is selected when the spacebar/enter key is pressed for a focused tab.
 
-Screen readers will read each tab's content when moused over similar to button.
-The ARIA tablist and tab roles will inform the screen reader.
+The `TAB` key moves focus out of the tabs.
 
 ## Accessibility
 
+ARIA provides tab-specific roles:
+
+- [tab](https://www.w3.org/TR/wai-aria-1.1/#tab): A grouping label providing a mechanism for selecting the tab content that is to be rendered to the user.
+- [tablist](https://www.w3.org/TR/wai-aria-1.1/#tablist): A list of tab elements, which are references to tabpanel elements.
+- [tabpanel](https://www.w3.org/TR/wai-aria-1.1/#tabpanel): A container for the resources associated with a tab, where each tab is contained in a tablist.
+
 TabList and Tab will apply the `tablist` and `tab` ARIA roles.
-See the behaviors for selection, focus trap, and programmatic accessibility.
+The `tabpanel` will be reserved for possible future component.
+
+### Screen readers
+
+Screen readers will read each tab's content.
+
+# Open issues
+
+## Should this component support overflow?
```

Comment:
Since we've decided to not include this in the initial implementation at least we should probably indicate this decision in here.